include/caf/leds.rst

```diff
@@ -18,19 +18,18 @@ To use the module, you must enable the following Kconfig options:
 * :option:`CONFIG_CAF_LEDS` - This option enables the LEDs module.
 * :option:`CONFIG_PWM` - This option enables Zephyr's :ref:`zephyr:pwm_api` driver, which is used for setting the LED color (that is, the brightness of the diodes).
   Setting this option requires you to define PWM ports, as described in the following section.
+* :option:`CONFIG_LED` - This option enables the LED driver API.
+* :option:`CONFIG_LED_PWM` - This option enabled the LED PWM driver.
+  Setting this option requires you to define PWM LEDs, as described in the following section.
```

Comment:
The section was renamed, so the reference to PWM is unclear now.

Reply:
as described above I can point to a section point